user/sorghum-lines-genomics-year-2.md

```diff
@@ -0,0 +1,196 @@
+# Sorghum Lines for Genomics Experiment \(Year 2\)
+
+| **accession** | [**taxid**](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=4558) | [**organism common name**](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=4558) | **subspecific genetic lineage rank** | **subspecific genetic lineage name** | [**ploidy**](https://phytozome.jgi.doe.gov/pz/portal.html#!info?alias=Org_Sbicolor) | [**number of replicons**](https://phytozome.jgi.doe.gov/pz/portal.html#!info?alias=Org_Sbicolor) | [**estimated size**](https://phytozome.jgi.doe.gov/pz/portal.html#!info?alias=Org_Sbicolor) | [**source material identifiers**](https://www.genesys-pgr.org/) | **sample name** |
```

Comment:
Here's the full table: https://docs.google.com/spreadsheets/d/1ntnps0u0c8O4xxrv_VVqfxDkpmppCGdFIIWLMmRCeSM/edit?usp=sharing